### ISSUE TITLE
Building release repos on travis

### ISSUE THREAD

**Issue by asmodehn** [pyros-dev/catkin_pip#17]
We need to investigate if/how this is feasible.

The idea is that we could use [travis](https://travis-ci.org), even for [third-party packages](http://wiki.ros.org/bloom/Tutorials/ReleaseThirdParty) (especially useful for dependencies coming from pure python packages)

This is required before we can extract CMakeLists.txt and packages.xml from catkin-pip (pyros) packages repository, and mutate them in third party packages...

**Comment by asmodehn** [comment]
This is already done for a few pyros related packages. And we have made a few thrid party packages already.
